week-1/exercises/decimal.js

```diff
@@ -3,8 +3,16 @@ const num = 56.5467;
 
 // You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
 
-// Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
-// Create a variable called decimalPart and assign to it an expression that evaluates to 0.5467 ( the decimal part of num )
-// Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
+// Create a variable called wholeNumberPart and assign the whole number part of num to it
+const wholeNumberPart = Math.floor(num);
+
+// Create a variable called decimalPart and assign the decimal part of num to it
+const decimalPart = num - wholeNumberPart;
```

Comment:
There are some quirks in the nature of JavaScript. You can take a look at this article to learn more about them. But your thinking is correct 👍 
https://www.avioconsulting.com/blog/overcoming-javascript-numeric-precision-issues